.nvmrc

```diff
@@ -0,0 +1 @@
+v4.2.1
```

Comment:
Warning, some lib (at least `swifter`) are not compiling on the latest OSX with that version of nodejs. Still investigating.